### DISCUSSION TITLE
How to add a heading inside <Steps> without adding a new step but still appear in the ToC?

### DISCUSSION THREAD

**Discussion by philipp-serfling** [shuding/nextra#3693] [Q&A]
After [this change](https://github.com/shuding/nextra/pull/3237) every heading will be converted into a step.

What I want:
- show h3 in the ToC and as a step
- show h4 in the ToC but not as a step

Code
```
<Steps>
### Download
#### MacOS/Linux
Install tool with [homebrew](https://www.google.com/)

#### Manual Installation
Download yourself

### Setup a container
</Steps>
```

Before:
h3 were treated as a step and the h4 were just rendered bigger. Both appeared in the ToC

![image](https://github.com/user-attachments/assets/eaef3960-bedc-43d3-ad50-9ccba687aded)

Now:
Every heading creates a new step which breaks hierachy
![image](https://github.com/user-attachments/assets/41092587-fb84-4192-bceb-5e3017317f26)

Any help much appreciated 🙏

**Comment by dimaMachina** [comment]
https://nextra.site/docs/guide/built-ins/steps#excluding-headings-from-table-of-contents

**Reply by philipp-serfling** [reply]
Thx for the quick reply!

But I wan't exactly the opposite!
I want the headings to **do** appear in the ToC and  **do** respect the hierachy and **not** to appear as a first class citizen Step with a number.

@dimaMachina Any idea?

**Reply by dimaMachina** [reply]
That makes sense, you need to tweak out this style https://github.com/shuding/nextra/blob/main/packages/nextra/styles/steps.css

PR Welcome
